jomar-restrepo/tarea2/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html semantico</title>
+    <link rel="stylesheet" href="/jomar-restrepo/tarea2/styles/styles.css" />
```

Comment:
@Leviathan9020 , you should use realtive paths
```suggestion
    <link rel="stylesheet" href="styles/styles.css" />
```